ports/rp2/machine_uart.c

```diff
@@ -72,6 +73,21 @@
 #define UART_HWCONTROL_CTS  (1)
 #define UART_HWCONTROL_RTS  (2)
 
+STATIC mutex_t write_mutex_0;
+STATIC mutex_t write_mutex_1;
+
+STATIC inline void write_mutex_init(mutex_t *m) {
```

Comment:
since these look like convenience functions, maybe it's cleaner if they take a `machine_uart_obj_t *` as their argument?